package.json

```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "xo",
-    "build": "cd swift && xcodebuild && mv build/release/aperture main && rm -r build",
-    "postinstall": "npm run build"
+    "build-macos": "cd swift && xcodebuild && mv build/release/aperture main && rm -r build",
+    "postinstall": "case $OSTYPE in darwin*) npm run build-macos ;; esac"
```

Comment:
I don't think we care about that, but you easily do `[ "$(uname)" = "Darwin" ] && npm run build-macos`  then.